components/button/button-icon.js

```diff
@@ -117,7 +118,8 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					box-shadow: var(--d2l-button-icon-focus-box-shadow);
 				}
 
-				.d2l-button-icon {
+				d2l-icon,
```

Comment:
Optionally, we can lock the slot down to prevent people from stuffing things inside it we don't expect with some CSS along these lines:
```css
slot[name="icon"]::slotted(*) {
    display: none;
}
slot[name="icon"]::slotted(d2l-icon-custom) {
    display: inline-block; /* or whatever display value it would normally be */
    height: 0.9rem;
    width: 0.9rem;
}
```
Not sure if locking down was discussed. I think we've done it elsewhere a couple times.

Reply:
That's a good idea. Then if there happen to be other things people want to put in there we would be able to evaluate on a case by case basis